Sources/include/ALMessage.h

```diff
@@ -25,6 +25,8 @@ static int const ALMESSAGE_CONTENT_CAMERA_RECORDING = 9;
 static int const ALMESSAGE_CHANNEL_NOTIFICATION = 10;
 static int const ALMESSAGE_CONTENT_CUSTOM = 101;
 static int const ALMESSAGE_CONTENT_HIDDEN = 11;
+static int const ALMESSAGE_CONTENT_INITIAL_STATIC_MESSAGE = 102;
```

Comment:
We've created a new message type for this?

Reply:
yes.but only on UI side